app/src/main/java/org/elasticsearch/bootstrap/BootstrapProxy.java

```diff
@@ -187,7 +187,7 @@ private void setup(boolean addShutdownHook, Environment environment) throws Boot
         }
 
         /*
-         * DISABLED setup of security manager due to policy problems with plugins (e.g. SigarPlugin will not work)
+         * DISABLED setup of security manager due to policy problems with plugins and dependencies.
```

Comment:
Still doesn't work?

Reply:
no still does not work. there is an error in some dependencies, haven't had a closer look yet.